### PR TITLE
Update FR translation

### PR DESCRIPTION
**Pull request by BjkGkh**

### CODE REVIEW

File: PoGo.NecroBot.CLI/Config/Translations/translation.fr.json [hunk 1]
```diff
@@ -119,6 +119,219 @@
...
+      "Key": "logEntryUpdate",
+      "Value": "UPDATE"
+    },
+    {
+      "Key": "loggingIn",
+      "Value": "Connexion via {0}"
+    },
+    {
+      "Key": "ptcOffline",
+      "Value": "Les serveurs PTC semblent hors-ligne ou vos identifiants PTC sont incorrects. Merci d'essayer via Google."
```
Comment (KurtCocain) [inline PoGo.NecroBot.CLI/Config/Translations/translation.fr.json]: ''Merci d'essayer via google'' Means <thanks> for trying with google

Please - > s'il vous plaît essayer via google.

Or imperative - > Essayez (not essayer) via google

**Comment by KurtCocain** [comment]
''Merci d'essayer via google'' Means "thanks" for trying with google

Please - > s'il vous plaît essayer via google.

Or imperative - > Essayez (not essayer) via google

I see that a lot of the translation goes like this...

''Thanks for trying this"
"Thanks for doing that"

We don't say it like that in french, it is not grammatically correct.
We use imperative ( Which isn't rude btw, used a lot in french)
Or we ask please, we don't give thanks.

**Comment by jayjayNC** [comment]
Of course it is correct, we do use "Merci d'essayer via..".

For example "Merci d'essayer à nouveau plus tard" > Please try again later.

"Please - > s'il vous plaît essayer via google." does not sound natural at all. does not sound like a ntavie speaker at all.

Translation does not need to be changed.

**Comment by GTXMike** [comment]
where i can get english file to translate to polish? file in bot EN is empty :/

**Comment by yamashi** [comment]
Je confirme, la formulation ne me derange pas

**Pull request merged**